### PR TITLE
Release 2.2.1

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by rnelson0** [comment]
We shouldn't need to delete .pmtignore to trigger the tag. It may also bloat the package size, which is why it's usually present. Was that issue fixed in the PMT?

**Comment by bastelfreak** [comment]
the puppet doc says that it honors the .gitignore or .pmtignore. the latter one is preffered if both files are present. travis installs gems into vendor/, our gitignore handles this, but the pmtignore doesn't. these files are redundant, but gitignore is handled by modulesync. .

**Comment by rnelson0** [comment]
That's the bug, it wasn't honoring it. If you can replicate what CI does
and it's not oversized, go for it, otherwise I'll look up the bug ID
 tomorrow.


-- 

Rob Nelson
rnelson0@gmail.com

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 2016-05-08 Release 2.2.1
+
+* delete the .pmtignore to trigger forge upload
```
Comment (rnelson0) [inline CHANGELOG.md]: We're deleting it because it's no longer needed, now!

**Pull request merged**